daemon/images.go

```diff
@@ -81,18 +86,28 @@ func (d *Daemon) pollForNewImages(logger log.Logger) {
 	}
 }
 
-// getUnlockedAutomatedServicesPolicyMap returns a resource policy map for all unlocked automated services
-func (d *Daemon) getUnlockedAutomatedServicesPolicyMap(ctx context.Context) (policy.ResourceMap, error) {
-	var services policy.ResourceMap
-	err := d.WithClone(ctx, func(checkout *git.Checkout) error {
-		var err error
-		services, err = d.Manifests.ServicesWithPolicies(checkout.ManifestDir())
-		return err
-	})
+type resources map[flux.ResourceID]resource.Resource
+
+func (r resources) IDs() (ids []flux.ResourceID) {
+	for k, _ := range r {
+		ids = append(ids, k)
+	}
+	return ids
+}
+
+// getUnlockedAutomatedServices returns all the resources that are
```

Comment:
the suspension is real!

Reply:
:joy: owps, will try harder to complete my sentences.